dnsmessage.proto

```diff
@@ -41,45 +41,47 @@ message PBDNSMessage {
     TCP = 2;                                    // Transmission Control Protocol (RFC 793)
   }
   enum PolicyType {
-    UNKNOWN = 1;                                // No policy applied, or unknown type
+    UNKNOWN = 1;                                // No RPZ policy applied, or unknown type
```

Comment:
We used to be careful about not tying our policy code to RPZ too much, so perhaps we should only list RPZ as an example of such policies?

Reply:
After discussing it with Konrad, we currently don't have any case where this field is set for anything else than a RPZ hit, so let's keep it that way. In theory it could be set from Lua (although the FFI bindings for that are missing) so perhaps it would be better to add some sort of "PolicySource" field that would indicate whether it was a RPZ, Lua, or something else.